### DISCUSSION TITLE
Very frequent "CI agent stopped responding!" build failures

### DISCUSSION THREAD

**Discussion by asomers** [cirruslabs/cirrus-ci-docs#1231] [General]
The [Mockall](https://github.com/asomers/mockall) project includes a nightly cron build to ensure that the project works with the latest Rust compiler.  Beginning on 30-August, this cron build has failed almost every time (7 out of 8) because of errors like the following.  If I restart the build, it usually succeeds.  But sometimes I must restart it multiple times.  Interestingly, of the 3 tasks only the "nightly" task fails in this way.  That's the slowest task.  It usually takes about 5.5 minutes.  Is there any way to figure out why the task times out so often?
```
CI agent stopped responding!
Container errored with 'Pod failed!'
```
Link to an example failed task: https://cirrus-ci.com/task/6088463120859136 .

**Comment by fkorotkov** [comment]
This should be fixed as of 9am EST today. Sorry for the temporary inconveniences.
